database/migrations/postgres/1720455590920137_resize_environments_json.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE environments ALTER COLUMN json TYPE varchar(1000000);
```

Comment:
Why should we put a limit on the number of characters? we can use `VARCHAR` without any limits, like similar columns in other tables.
```suggestion
ALTER TABLE environments ALTER COLUMN json TYPE VARCHAR;
```